hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -687,27 +689,47 @@ func (r *NodePoolReconciler) reconcile(ctx context.Context, hcluster *hyperv1.Ho
 		return ctrl.Result{RequeueAfter: duration}, nil
 	}
 
-	tuningConfigMap := TuningConfigMap(controlPlaneNamespace, nodePool.Name)
-	if tuningConfig == "" {
-		err = r.Get(ctx, client.ObjectKeyFromObject(tuningConfigMap), tuningConfigMap)
+	tunedConfigMap := TunedConfigMap(controlPlaneNamespace, nodePool.Name)
+	if tunedConfig == "" {
+		err = r.Get(ctx, client.ObjectKeyFromObject(tunedConfigMap), tunedConfigMap)
 		if err != nil && !apierrors.IsNotFound(err) {
-			return ctrl.Result{}, fmt.Errorf("failed to get tuningConfig ConfigMap: %w", err)
+			return ctrl.Result{}, fmt.Errorf("failed to get tunedConfig ConfigMap: %w", err)
 		}
 		if err == nil {
-			if err := r.Delete(ctx, tuningConfigMap); err != nil && !apierrors.IsNotFound(err) {
-				return ctrl.Result{}, fmt.Errorf("failed to delete tuningConfig ConfigMap with no Tuneds defined: %w", err)
+			if err := r.Delete(ctx, tunedConfigMap); err != nil && !apierrors.IsNotFound(err) {
+				return ctrl.Result{}, fmt.Errorf("failed to delete tunedConfig ConfigMap with no Tuneds defined: %w", err)
 			}
 		}
 	} else {
-		if result, err := r.CreateOrUpdate(ctx, r.Client, tuningConfigMap, func() error {
-			return reconcileTuningConfigMap(tuningConfigMap, nodePool, tuningConfig)
+		if result, err := r.CreateOrUpdate(ctx, r.Client, tunedConfigMap, func() error {
+			return reconcileTunedConfigMap(tunedConfigMap, nodePool, tunedConfig)
 		}); err != nil {
 			return ctrl.Result{}, fmt.Errorf("failed to reconcile Tuned ConfigMap: %w", err)
 		} else {
 			log.Info("Reconciled Tuned ConfigMap", "result", result)
 		}
 	}
 
+	performanceProfileConfigMap := PerformanceProfileConfigMap(controlPlaneNamespace, nodePool.Name)
```

Comment:
can we use performanceProfile or pp across de the code base consistently everywhere? I value clear descriptive names over brevity so my bias is towards the first one.